app/assets/stylesheets/application.css

```diff
@@ -12,4 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ //= require stylesheets/pure.0.5.0.grids-responsive-min
```

Comment:
I replaced these with:
*= require ../../../vendor/assets/stylesheets/pure.0.5.0.grids-responsive-min
 *= require /../../vendor/assets/stylesheets/pure.0.5.0.pure-min

..and the page loaded. But no visible style had changed. Everything was still vertically aligned.